packages/at_secondary_server/lib/src/verb/metrics/metrics_impl.dart

```diff
@@ -479,3 +481,27 @@ class NotificationCompactionStats implements MetricProvider {
     return 'NotificationCompactionStats';
   }
 }
+
+class LatestCommitEntryOfEachKey implements MetricProvider {
+  @override
+  getMetrics({String? regex}) async {
+    var responseMap = <String, Map<String, dynamic>>{};
+    var atCommitLog = await (AtCommitLogManagerImpl.getInstance()
+        .getCommitLog(AtSecondaryServerImpl.getInstance().currentAtSign));
+
+    var commitEntryIterator = atCommitLog!.getEntries(-1);
+
+    while (commitEntryIterator.moveNext()) {
+      responseMap[commitEntryIterator.current.key] = {
+        'commitId': commitEntryIterator.current.value.commitId,
+        'commitOp': commitEntryIterator.current.value.operation.toString()
```

Comment:
Just a suggestion, but for brevity in the response you could consider having an array rather than an object, and using the symbol for the op - for example instead of this
```json
{
  "@bob:some_thing.some_app@alice":{"commitId":101,"commitOp":"UPDATE"},
  "public:firstname.wavi@alice":{"commitId":105,"commitOp":"UPDATE_ALL"},
  "@bob:some_other_thing.some_app@alice":{"commitId":112,"commitOp":"DELETE"}
}
```
the response could instead look like
```json
{
  "@bob:some_thing.some_app@alice":[101,"+"],
  "public:firstname.wavi@alice":[105,"*"],
  "@bob:some_other_thing.some_app@alice":[112,"-"],
}
```

Reply:
Sure Gary, Will make the changes.